### ISSUE TITLE
Users should be able to access a hidden skip nav link at the top of each page to jump to the ask question box.

### ISSUE THREAD

**Issue by MarkGOC** [dto-btn/ssc-assistant#580]
We need to add a hidden skip link for accessibility

This will allow keyboard users to jump to our message box to submit a question to SSCA.

Video on the idea
https://www.youtube.com/watch?v=jDDaOFr9nqQ

We should look at the GOC design system and templates to see how they do it.  In the past we used the public templates for internal apps to make our design lives easier.

Problem is I like our Material Design, Design system.  so we might want to just make it accessible.  And we need to make it work in React
